src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -672,22 +672,24 @@ bool SolverBase::lineSearchWithParabolicInterpolation( real64 const & time_n,
  */
 real64 SolverBase::eisenstatWalker( real64 const newNewtonNorm,
                                     real64 const oldNewtonNorm,
-                                    real64 const weakestTol )
+                                    LinearSolverParameters::Krylov const & krylovParams,
+                                    integer const logLevel )
 {
-  real64 const strongestTol = 1e-8;
-  real64 const exponent = 2.0;
-  real64 const gamma = 0.9;
-
-  real64 normRatio = newNewtonNorm / oldNewtonNorm;
-  if( normRatio > 1 )
-    normRatio = 1;
-
-  real64 newKrylovTol = gamma*std::pow( normRatio, exponent );
-  real64 altKrylovTol = gamma*std::pow( oldNewtonNorm, exponent );
```

Comment:
this one is confusing, i replaced `oldNewtonNorm` but old linear solver tolerance, but not quite sure about that